app/views/hello_world/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1 class="alert alert-info this-works">Hello World</h1>
```

Comment:
@nostophilia 

There should be no class values on this line.

<h1>Hello World</h1>